neonvm/runner/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apk add --no-cache \
 WORKDIR /workspace
 # FIXME: There's non-overlapping version support for <1.27 and >=1.27.
 # We should carefully consider how we go about future-proofing this (or not).
-ENV VERSION="v1.26.1"
+ENV VERSION="v1.27.1"
```

Comment:
We can remove the comment above this now, right?

Reply:
Maybe we remove the configuration option to enable `crictl`, so that it won't accidentally enabled by someone (e.g. by taking a release yaml without autoscaling-deploy's patches), since we expect it not to work with 1.27+?